web_src/js/features/repo-legacy.ts

```diff
@@ -54,6 +54,7 @@ export function initRepository() {
   initRepoCloneButtons();
   initCitationFileCopyContent();
   initRepoSettings();
+  initRepoIssueWipNewTitle();
```

Comment:
Why are we adding more code to `repo-legacy`?
Based on the name, I would have assumed that this is a delete-only file.

Reply:
Because it was in `index.ts`: `initRepoIssueWipTitle`.

At the moment there is no proper place to call it. Putting it in the `initRepository` of `repo-legacy` is still better than before.